python/nano/src/bigdl/nano/pytorch/trainer/Trainer.py

```diff
@@ -367,6 +368,8 @@ def trace(model: nn.Module,
                             backend. 'openvino' and 'onnxruntime' are supported for now.
         :param onnxruntime_session_options: The session option for onnxruntime, only valid when
                                             accelerator='onnxruntime', otherwise will be ignored.
+        :param opset_version: opset_version for export_to_onnx, only valid when
+                                            accelerator='onnxruntime', otherwise will be ignored.
```

Comment:
I think `accelerator="openvino"` will also be affected by this opset_version? no? @zhentaocc

Reply:
I suppose highest opset version should work.